libsql-sqlite3/src/main.c

```diff
@@ -1238,6 +1238,9 @@ static int sqlite3Close(sqlite3 *db, int forceZombie){
   */
   sqlite3VtabRollback(db);
 
+  /* Destroy the create wal */
+  /* (db->create_wal.xDestroy)(db->create_wal.self); */
```

Comment:
Is it commented out because it's a draft and it's going to be back later? Or are we changing the way wal is destroyed? In any case, we need a FIXME/TODO comment placed here

Reply:
This is the quirk I wanted to talk about. With shared pagers, the ownership of the create_wal is shared as well, because pagers are lifted to the global scope. So the connection does not necessarily owns the create_wal. This is only ok for now because there is nothing to clean with the `sqlite3_create_wal`. I think we need to have that reference counted, but maybe this is done another way in C?